src/server.rs

```diff
@@ -90,7 +90,32 @@ impl ConnectionInfo {
 ///     .serve(MyService { count: Mutex::new(0) })
 ///     .expect("failed to start server");
 /// ```
-pub trait Service: Send + Sync + 'static {
+///
+/// or
```

Comment:
Might be helpful to point out the difference here.

```suggestion
/// If your service is already cheaply cloneable, you can instead use `serve_clone` and avoid an extra `Arc` wrapper:
```